### ISSUE TITLE
Don't suggest while ... if as a one-liner

### ISSUE THREAD

**Issue by forthrin** [rubocop/rubocop#14971]
## Steps to reproduce the problem

```ruby
while $stdin.read
  warn 'Breaking up with Taylor Swift may have unforeseen consequences' if Time.now.strftime('%S').to_i.odd?
end
```

## Expected behavior

**No cop.** Assuming `(warn if) while` is the supposed rewrite, which appears unclear and clunky.

## Actual behavior

`WhileUntilModifier`

## RuboCop version

```
ruby 4.0.1 (2026-01-13 revision e04267a14b) +PRISM [arm64-darwin25]
rubocop     2026-02-28 eb973f45f
```

**Comment by sorafujitani** [comment]
since I'm facing the same issues with this problem, I'll create a PR.
should be able to conduct the necessary discussions there.
